packages/devtools_app/lib/src/ui/tab.dart

```diff
@@ -165,8 +165,8 @@ class _AnalyticsTabbedViewState extends State<AnalyticsTabbedView>
 
   @override
   void dispose() {
-    _tabController?.removeListener(_onTabChanged);
-    _tabController?.dispose();
+    _tabController!.removeListener(_onTabChanged);
+    _tabController!.dispose();
```

Comment:
leave the ? null checks here instead of force unwrapping

Reply:
done